src/main/java/uk/gov/hmcts/reform/sscs/mappers/ServiceHearingValuesMapper.java

```diff
@@ -0,0 +1,118 @@
+package uk.gov.hmcts.reform.sscs.mappers;
```

Comment:
can you move this to 
package uk.gov.hmcts.reform.sscs.helper.mapping;

Reply:
Moving, next commit. Renaming class name to **Mapping to correspond to other members of same subdirectory.